mods/ra/languages/rules/en.ftl

```diff
@@ -678,146 +676,146 @@ actor-anypower-name = Any Power Generation
-   .name = Minelayer
+      Unarmed.
 
 actor-truk =
-   .description = Transports cash to other players.
-      Unarmed
    .name = Supply Truck
+   .description = Transports cash to other players.
+      Unarmed.
 
 actor-mgg =
-   .description = Regenerates the shroud nearby,
-    obscuring the area.
-      Unarmed
    .name = Mobile Gap Generator
+   .description = Regenerates shroud nearby,
+    obscuring the area.
+      Unarmed.
 
 actor-mrj =
    .name = Mobile Radar Jammer
-   .description = Jams nearby enemy radar domes
+   .description = Jams nearby enemy Radar Domes
     and deflects incoming missiles.
-      Unarmed
+      Unarmed.
 
 actor-ttnk =
-   .description = Tank with mounted Tesla coil.
-      Strong vs Infantry, Vehicles, Buildings
-      Weak vs Aircraft
    .name = Tesla Tank
    .generic-name = Tank
+   .description = Tank with mounted Tesla coil.
+      Strong vs. Infantry, Vehicles and Buildings.
+      Weak vs. Aircraft.
 
 actor-ftrk =
-   .description = Mobile unit with mounted Flak cannon.
-      Strong vs Infantry, Light armor, Aircraft
-      Weak vs Tanks
    .name = Mobile Flak
+   .description = Mobile unit with a Flak cannon.
+      Strong vs. Infantry, Light armor and Aircraft.
+      Weak vs. Tanks.
 
 actor-dtrk =
-   .description = Truck with actively armed nuclear
-    explosives. Has very weak armor.
    .name = Demolition Truck
+   .description = Truck carrying armed nuclear explosives,
+    with very weak armor.
 
 actor-ctnk =
-   .description = Armed with anti-ground missiles.
-    Teleports to areas within range.
-      Strong vs Vehicles, Buildings
-      Weak vs Infantry, Aircraft
-      Special ability: Can teleport
    .name = Chrono Tank
    .generic-name = Tank
+   .description = Armed with anti-ground missiles,
+    and capable of teleporting to any area within range.
+      Strong vs. Vehicles and Buildings.
+      Weak vs. Infantry and Aircraft.
+      Special ability: Can teleport.
 
 actor-qtnk =
-   .description = Deals seismic damage to nearby vehicles
-    and structures.
-      Strong vs Vehicles, Buildings
-      Weak vs Infantry, Aircraft
    .name = MAD Tank
    .generic-name = Tank
+   .description = Deals seismic damage to nearby vehicles
+    and structures.
+      Strong vs. Vehicles and Buildings.
+      Weak vs. Infantry and Aircraft.
 
 actor-stnk =
-   .description = Lightly armored infantry transport which
-    can cloak. Armed with anti-ground missiles.
-      Strong vs Light armor
-      Weak vs Infantry, Tanks, Aircraft
    .name = Phase Transport
+   .description = Light armored infantry transport which
+    can cloak. Armed with anti-ground missiles.
+      Strong vs. Light armor.
+      Weak vs. Infantry, Tanks and Aircraft.
 
 ## Civilian Tech
 actor-hosp =
    .name = Hospital
    .captured-desc = Provides infantry with self-healing.
-   .capturable-desc = Capture to enable self-healing for infantry.
+   .capturable-desc = Capture to let infantry self-heal.
```

Comment:
The `capturable-desc`'s here haven't been updated consistently as with other mods.